grafana-plugin/e2e-tests/integrations/maintenanceMode.test.ts

```diff
@@ -135,6 +129,7 @@ test.describe('maintenance mode works', () => {
       userName,
       'Maintenance'
     );
+    test.slow();
```

Comment:
does it make a difference if `test.slow()` is called before `createIntegrationAndEscalationChainAndEnableMaintenanceMode` or not?

Reply:
I think it doesn't make a difference and `test.slow()` by convention should be placed at the very beginning of the test. So that's a good catch, thanks! I'll move it up